### PR TITLE
ENCD-5673-add-scss-linter

### PR DESCRIPTION
**Pull request by zoldello**

### CODE REVIEW

File: src/encoded/static/scss/encoded/modules/_characterizations.scss [hunk 1]
```diff
@@ -512,7 +511,8 @@ $detail-switch-width: 30px;
     left: 0;
     width: 100%;
     height: 40px;
-    background-image: linear-gradient(top,  rgba(0,0,0,0) 0%,rgba(0,0,0,0.3) 100%);
+    /* stylelint-disable-next-line function-linear-gradient-no-nonstandard-direction */
+    background-image: linear-gradient(top, rgba(0, 0, 0, 0) 0%, rgba(0, 0, 0, 0.3) 100%);
```
Comment (forresttanaka) [inline src/encoded/static/scss/encoded/modules/_characterizations.scss:515]: Huh, never knew that was invalid.

File: src/encoded/static/scss/encoded/modules/_loading-spinner.scss [hunk 2]
```diff
@@ -1,56 +1,56 @@
...
 }
 
 .communicating--centered {
-	.loading-spinner--centered {
-    	display: flex;
-    	height: 10px // only here to get IE11 to work, see: https://exceptionshub.com/flexbox-not-working-in-internet-explorer-11.html
-	}
+    .loading-spinner--centered {
+        display: flex;
+        height: 10px // only here to get IE11 to work, see: https://exceptionshub.com/flexbox-not-working-in-internet-explorer-11.html
```
Comment (forresttanaka) [inline src/encoded/static/scss/encoded/modules/_loading-spinner.scss:54]: Strange; I wonder why it didn’t ding this missing semicolon?

File: src/encoded/static/scss/encoded/modules/_status.scss [hunk 3]
```diff
@@ -1,5 +1,6 @@
 // All status indicator colors. As statuses change in the schemas, should see if corresponding
 // changes are needed here.
+/* stylelint-disable-next-line value-list-max-empty-lines */
```
Comment (forresttanaka) [inline src/encoded/static/scss/encoded/modules/_status.scss:3]: Did it just not like the blank lines before the comment lines?

**Review by forresttanaka — DISMISSED** [review]
These new rules look really good to me. I had no idea we had so many usages of max-width — we used to have zero. We’ll need to put in some effort to get rid of all of those over time.

**Comment by zoldello** [comment]
fixed a merge conflict after "QA Completed"

**Review by forresttanaka — DISMISSED** [review]

**Review by forresttanaka — APPROVED** [review]
I reviewed _pipeline.scss and it looks good.

**Pull request merged**